tools/installers/install_chime7task1.sh

```diff
@@ -0,0 +1,10 @@
+./install_s3prl.sh
+pip install git+https://github.com/popcornell/lhotse@master
```

Comment:
Is your version of Lhotse different from the trunk?

Reply:
yes, i have a PR for exporting to multi channel kaldi manifests

Reply:
okay let's see if we can get that checked in ASAP.

Reply:
This can be changed now that the PR is merged.

Reply:
Yeah going to change it now